### PR TITLE
Bump GCI to gci-stable-56-9000-84-2 on release-1.5

### PR DESCRIPTION
**Pull request by freehan**
Manual cherry-pick of https://github.com/kubernetes/kubernetes/pull/41819

### CODE REVIEW

File: cluster/kubemark/config-default.sh [hunk 1]
```diff
@@ -32,9 +32,9 @@ MASTER_ROOT_DISK_SIZE=${KUBEMARK_MASTER_ROOT_DISK_SIZE:-10GB}
 REGISTER_MASTER_KUBELET=${REGISTER_MASTER:-false}
 PREEMPTIBLE_NODE=${PREEMPTIBLE_NODE:-false}
 
-MASTER_OS_DISTRIBUTION=${KUBE_MASTER_OS_DISTRIBUTION:-debian}
+MASTER_OS_DISTRIBUTION=${KUBE_MASTER_OS_DISTRIBUTION:-gci}
```
Comment (freehan) [inline cluster/kubemark/config-default.sh]: I noticed that kubemark in 1.5 branch is not using gci image as master. Not sure what is the reason behind it. I change it to match the config of master.

Comment (dchen1107) [inline cluster/kubemark/config-default.sh]: I think this should be fine, especially on HEAD, it is switched to GCI / COS already. cc/ @gmarek @wojtek-t for confirmation.

**Comment by dchen1107** [comment]
/lgtm and
/appove

cc/ @saad-ali the release czar

**Comment by saad-ali** [comment]
+@mwielgus the new 1.5 release branch manager: This is worth keeping a close eye on. After the previous GCI image update, a bunch of 1.5 CI broke.

**Comment by freehan** [comment]
@k8s-bot unit test this
@k8s-bot kubemark e2e test this

**Comment by freehan** [comment]
adding labels based on comment

**Comment by freehan** [comment]
@k8s-bot kubemark e2e test this

**Comment by freehan** [comment]
@k8s-bot kubemark e2e test this

**Comment by freehan** [comment]
Looks like kubemark in 1.5 is not compatible with GCI image. Will switch back to use CVM for master


```
W0222 11:15:28.930] mkdir: cannot create directory '/srv': Read-only file system
W0222 11:15:28.986] !!! Error in ./test/kubemark/start-kubemark.sh:150
W0222 11:15:28.987]   'gcloud compute ssh --zone="${ZONE}" --project="${PROJECT}" "${MASTER_NAME}" --command="sudo mkdir /srv/kubernetes -p &&     sudo bash -c \"echo ${MASTER_CERT_BASE64} | base64 --decode > /srv/kubernetes/server.cert\" &&     sudo bash -c \"echo ${MASTER_KEY_BASE64} | base64 --decode > /srv/kubernetes/server.key\" &&     sudo bash -c \"echo ${CA_CERT_BASE64} | base64 --decode > /srv/kubernetes/ca.crt\" &&     sudo bash -c \"echo ${KUBECFG_CERT_BASE64} | base64 --decode > /srv/kubernetes/kubecfg.crt\" &&     sudo bash -c \"echo ${KUBECFG_KEY_BASE64} | base64 --decode > /srv/kubernetes/kubecfg.key\" &&     sudo bash -c \"echo \"${KUBE_BEARER_TOKEN},admin,admin\" > /srv/kubernetes/known_tokens.csv\" &&     sudo bash -c \"echo \"${KUBELET_TOKEN},kubelet,kubelet\" >> /srv/kubernetes/known_tokens.csv\" &&     sudo bash -c \"echo \"${KUBE_PROXY_TOKEN},kube_proxy,kube_proxy\" >> /srv/kubernetes/known_tokens.csv\" &&     sudo bash -c \"echo ${password},admin,admin > /srv/kubernetes/basic_auth.csv\""' exited with status 1
W0222 11:15:28.987] Call stack:
W0222 11:15:28.987]   1: ./test/kubemark/start-kubemark.sh:150 main(...)
W0222 11:15:28.987] Exiting with status 1
```

**Comment by freehan** [comment]
@k8s-bot test this

**Comment by freehan** [comment]
@k8s-bot cross build this
@k8s-bot kubemark e2e test this
@k8s-bot aws e2e test this
@k8s-bot non-cri node e2e test this

**Pull request merged**